frontend/packages/console-app/src/components/cloud-shell/CloudShellTab.scss

```diff
@@ -1,9 +1,11 @@
 .co-cloud-shell-tab {
-  display: flex;
-  flex-direction: column;
-  width: 100%;
-  height: 100%;
+  position: fixed;
+  top: 0px;
+  left: 0px;
+  right: 0px;
+  bottom: 0px;
```

Comment:
Why is any of this necessary?
What's wrong with using the flex layout?

Reply:
It keeps expanding beyond the screen as user puts some command